examples/gaussian_process/new/plot_gpc_xor.py

```diff
@@ -0,0 +1,41 @@
+"""Gaussian process classification (GPC)
+
+This example illustrates GPC on XOR data.
+"""
+print __doc__
+
+# Authors: Jan Hendrik Metzen <jhm@informatik.uni-bremen.de>
+#
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.gaussian_process import GaussianProcessClassification
+from sklearn.gaussian_process.kernels import RBF
+
+xx, yy = np.meshgrid(np.linspace(-3, 3, 50),
+                     np.linspace(-3, 3, 50))
+np.random.seed(0)
+X = np.random.randn(200, 2)
+Y = np.logical_xor(X[:, 0] > 0, X[:, 1] > 0)
+
+# fit the model
+kernel = [1e-10, 1.0, 100] * RBF(param_space=(1e-10, 1.0, 10))
```

Comment:
Thanks, I found the definitions lower down. I like the overridden operators, but, as I indicate lower down, I am not 1000% sure about the param space definition. That said, I have nothing much better to offer right now, but will try to think.